core/src/main/java/org/axonframework/commandhandling/disruptor/CommandHandlerInvoker.java

```diff
@@ -118,10 +119,36 @@ public void onEvent(CommandHandlingEntry entry, long sequence, boolean endOfBatc
                                               AggregateFactory<T> aggregateFactory,
                                               SnapshotTriggerDefinition snapshotTriggerDefinition,
                                               ParameterResolverFactory parameterResolverFactory) {
+        return createRepository(eventStore,
+                                null,
```

Comment:
So it's okay for the `RepositoryProvider` to be `null` in several scenarios? How come?